### ISSUE TITLE
MySQL TLS deprecation test fails on 20.04

### ISSUE THREAD

**Issue by mz0** [mz0/mysql-5.7-ubuntu#1]
MySQL [5.7.35 deprecates TLSv1 and TLSv1.1](https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-35.html): `mysql` should warn client, and `mysqld` should log deprecated connections.

`t/ssl_deprecated_tls_versions.test` was added to check these warnings and server log, the warnings test passes, but log test fails completely on default 20.04, since Ubuntu denies that. See e.g. [1864689](https://bugs.launchpad.net/ubuntu/+source/openssl/+bug/1864689)

Following that bug advice I've changed `/etc/ssl/openssl.cnf`
```
diff -c /etc/ssl/openssl.cnf.0 /etc/ssl/openssl.cnf
*** /etc/ssl/openssl.cnf.0	2020-04-20 14:53:50.000000000 +0300
--- /etc/ssl/openssl.cnf	2021-07-29 09:43:09.658384062 +0300
***************
*** 1,3 ****
--- 1,4 ----
+ openssl_conf = mysqltest_conf
  #
  # OpenSSL example configuration file.
  # This is mostly being used for generation of certificate requests.
***************
*** 348,350 ****
--- 349,360 ----
  				# (optional, default: no)
  ess_cert_id_alg		= sha1	# algorithm to compute certificate
  				# identifier (optional, default: sha1)
+ ## temporary addition ##
+ [ mysqltest_conf ]
+ ssl_conf = ssl_sect
+ 
+ [ ssl_sect ]
+ system_default = system_default_sect
+ 
+ [ system_default_sect ]
+ CipherString = DEFAULT:@SECLEVEL=1
```
and re-checked
```
builddir/mysql-test$ ./mysql-test-run.pl --do-test=ssl_deprecated_tls_versions
Logging: ./mysql-test-run.pl  --do-test=ssl_deprecated_tls_versions
MySQL Version 5.7.35
Checking supported features...
 - SSL connections supported
Using suites: main,sys_vars,binlog,federated,gis,rpl,innodb,innodb_gis,innodb_fts,innodb_zip,innodb_undo,perfschema,funcs_1,opt_trace,parts,auth_sec,query_rewrite_plugins,gcol,sysschema,test_service_sql_api,json,connection_control
Collecting tests...
Checking leftover processes...
Removing old var directory...
Creating var directory '/home/mz0/m57/mysql-5.7/builddir/mysql-test/var'...
Installing system database...
Using parallel: 1

==============================================================================

TEST                                      RESULT   TIME (ms) or COMMENT
--------------------------------------------------------------------------

worker[1] Using MTR_BUILD_THREAD 300, with reserved ports 13000..13009
[ 50%] main.ssl_deprecated_tls_versions         [ pass ]     45
[100%] shutdown_report                          [ pass ]       
--------------------------------------------------------------------------
The servers were restarted 0 times
Spent 0.045 of 4 seconds executing testcases

Completed: All 2 tests were successful.
```
Consider disabling server log check in this test as a temporary workaround.

**Comment by mz0** [comment]
a [better option](https://askubuntu.com/a/1233456) for temporarily setting `CipherString = DEFAULT:@SECLEVEL=1` would be
```
export OPENSSL_CONF=/path/to/temporary/openssl.cnf
```

NB [note](https://bugs.launchpad.net/ubuntu/+source/openssl/+bug/1864689/comments/5) suggests
```
man SSL_CTX_get_security_level.3ssl
```
To see that one needs `sudo apt install libssl-doc`

**Comment by mz0** [comment]
fixed in u75's 5ccd44d5
https://github.com/mz0/u57/commit/5ccd44d560148a3f477712bcea6874380bb6344e
